doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/primitives/text/localization/TextLocalizationPage.swift

```diff
@@ -1,26 +1,49 @@
 import SwiftUI
 
 struct TextLocalizationPage: View {
-    var someString = "pen"
-    var object = LocalizedStringResource("pencil")
     
     var body: some View {
         ScrollView {
             VStack(spacing: 24) {
-                // Localized if possible
+                // Text's initializer accepts a LocalizedStringKey, which can be implicitly constructed from a string literal
+                // The value of the string is used for the development language localization
+                // Values for other languages are looked up in the default table using the string as key, if not found the value is used as is
                 Text("pencil")
                 
-                // Variables not localized by default
+                // The comment is shown in the localization table, useful for giving context to translators
```

Comment:
In this comment, you mentioned a "localization table" showing the "comment" mentioned in this `Text` view, i.e., the comment is placed in some kind of table at runtime.

However, down below, you reference the `.main` localization table, which suggests that the table exists before the code is compiled, not after, and not at runtime.

Can you clarify what you mean by "localization table"?

Reply:
I added a comment about table generation, let me know if it still isn't clear